swift/looker/Tests/lookerTests/baseTransportTests.swift

```diff
@@ -15,6 +15,7 @@ fileprivate let repoPath : String = testRootPath + "/../../"
 fileprivate let localIni : String = ProcessInfo.processInfo.environment["LOOKERSDK_INI"] ?? (repoPath + "looker.ini")
 
 
+@available(OSX 10.15, *)
```

Comment:
Interesting that these appeared/changed?

Reply:
It's because I pushed more initialization into `TestConfig()` that requires 10.15